save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/OrganizationView.kt

```diff
@@ -488,6 +489,9 @@ class OrganizationView : AbstractView<OrganizationProps, OrganizationViewState>(
                     errorMessage = "Failed to create contest: ${it.status} ${it.statusText}"
                 }
             }
+            updateContestCallback = { contest ->
+                deleteContest(contest)
```

Comment:
Why do you pass this method as a callback? It doesn't interact with `OrganizationView`'s state, so I don't see why it can be encapsulated entirely in `OrganizationContestMenu`

Moreover, you've chosen a very bad naming. `OrganizationContestMenu` uses `props.updateContestCallback` whenever `Delete` button is pressed, and only in a parent component you set `updateCallback` as `deleteContest`. This makes API of these components very confusing